app/views/citizens/legal_aid_applications/index.html.erb

```diff
@@ -1,13 +1,13 @@
 <%= page_template page_title: t('.heading_1'), back_link: :none do %>
   <p class="govuk-body">
-    <b> <%= t('.name') %></b>
+    <strong> <%= t('.name') %></strong>
```

Comment:
I have a couple questions about the use of `<strong>` tags.

I'm intrigued what the justification is from SonarCloud for these being "Code smells"?

The [`<b>` tag is a valid HTML element with its own purpose](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b).

Do we want to communicate to the user that all of these elements are important or urgent?

```html
This is some text that is <strong>seriously important!</strong>
```

Or maybe we want to draw attention to elements without indicating that they are especially important?

```html
This is <b>some text that isn't important</b>, but I want to draw attention to it.
```

Maybe we just want to style some text with a bold weight?

```html
This is <span class="govuk-!-font-weight-bold">some text I want to look bold</span>
```

I think it's important we are conscious about the the tags we use, and we should be writing semantic HTML wherever we can. 

I'm worried SonarCloud just says "\<strong> tag good" and we follow it blindly without proper consideration.

What do you think?